src/StochasticDiffEq.jl

```diff
@@ -20,7 +20,7 @@ module StochasticDiffEq
 
   using Logging
 
-  using LinearAlgebra, Random
+  using LinearAlgebra, Random, FillArrays
```

Comment:
Need to add FillArrays.jl to the Project.toml file.

Reply:
But Eye calls Diagonal. I not sure this will fix it.

Reply:
`Diagonal(n)` is not a base dispatch. It's a special FillArray thing that builds the representation without building the arrays of the matrix.

Reply:
Oh. Nice!